### ISSUE TITLE
Migrate all blitz package code into the nextjs fork

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#2945]
### What do you want and why?

In order to streamline nextjs upgrades, we need to move all blitz package code into the nextjs fork. Once that is finished, we can move the nextjs fork to the top level and then merge nextjs upgrades directly into this repo.

- [x] `packages/babel-preset` @flybayer https://github.com/blitz-js/blitz/pull/2903
- [x] `packages/display` (some or all already in next package) @beerose https://github.com/blitz-js/blitz/pull/2989
- [ ] `packages/installer`
- [ ] `packages/generator` - wait for https://github.com/blitz-js/blitz/pull/2679
- [ ] `packages/cli` 
  - [ ] Migrate all blitz cli commands into next cli, being careful to preserve existing behavior like auto PORT detection, etc
   - [ ] `packages/server` (will basically become obsolete once cli package is moved)
  - [ ] inside blitz pkg cli.ts, spawn `next` binary and pass all args
- [ ] `packages/blitz`
   -  Once this is removed, the `nextjs/packages/next` package becomes the main entry point for CLI and everything
   - Rename `nextjs/packages/next` to `blitz` in package.json name field only
   - Add babel plugin to rewrite imports from `next/*` to `blitz/*` for compatibility with people converting nextjs projects to blitz
- [ ] Migrate `test/integration` tests into `nextjs/test/integration/` 
- [ ] Everything else at top level. Some things like jest config might need removed completely or merged into corresponding config in `nextjs/`
- [ ] Move `nextjs` to top level

**Comment by imcodingideas** [comment]
I was thinking about that because with this coming: https://twitter.com/leeerob/status/1458846513451384832
